src/main/java/de/rabea/Game/Game.java

```diff
@@ -5,76 +5,49 @@
 public class Game {
 
     private final UserInterface userInterface;
-    private final ComputerPlayer computerPlayer;
+    private final Player player;
+    private final Player opponent ;
+    private final GameSetUp gameSetUp;
 
-    public Game(UserInterface userInterface, ComputerPlayer computerPlayer) {
+    public Game(UserInterface userInterface, Player player, Player opponent, GameSetUp gameSetUp) {
         this.userInterface = userInterface;
-        this.computerPlayer = computerPlayer;
+        this.player = player;
+        this.opponent = opponent;
+        this.gameSetUp = gameSetUp;
     }
 
     public void play() {
-        GameMode game = gameSetup();
-        Cell mark = Cell.X;
         Board board = new Board();
-        Rules rules = new Rules(board);
-        while (gameIsNotOver(rules)){
-            playOneHumanRound(mark, board);
-            mark = board.switchMark(mark);
-            if (humanVsComputerMode(game) && gameIsNotOver(rules)) {
-                playOneComputerRound(board, mark);
-                mark = board.switchMark(mark);
-            }
+        Player currentPlayer = player;
+        while (gameIsNotOver(board)){
+            playOneRound(currentPlayer, board);
+            currentPlayer = switchPlayer(currentPlayer);
         }
-        finishGame(mark, board, rules);
+        finishGame(currentPlayer.mark(), board);
```

Comment:
because you switch the player within the loop, when you do finish game, the mark will be of the opponent, so you need to switch back for the winning message to be displayed, which is a little unexpected (not too bad for this simple domain, but in a big project it may cause confusion).. 

You could pull the finishGame up into the loop, and check if the game is over before the players are switched?